lib/sycamore/sycamore/reader.py

```diff
@@ -291,22 +293,17 @@ def opensearch(
         )
 
         client_params = OpenSearchReaderClientParams(os_client_args=os_client_args)
-        query_params = (
-            OpenSearchReaderQueryParams(
-                index_name=index_name,
-                query=query,
-                reconstruct_document=reconstruct_document,
-                doc_reconstructor=doc_reconstructor,
-                kwargs=query_kwargs,
-            )
-            if query is not None
-            else OpenSearchReaderQueryParams(
-                index_name=index_name,
-                reconstruct_document=reconstruct_document,
-                doc_reconstructor=doc_reconstructor,
-                kwargs=query_kwargs,
-            )
+        if query is None:
```

Comment:
remove the default in OpenSearchReaderQueryParams
That change makes sure this code and the other code stay in sync.

Reply:
ok